package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A basic client side calculator",
   "main": "calculator.html",
   "scripts": {
-    "test": "node node_modules/mocha/bin/mocha spec --recursive --require spec/mocha-setup"
+    "test": "node node_modules/mocha/bin/mocha spec --recursive --require spec/mocha-setup",
+    "start": "node node_modules/http-server/bin/http-server"
```

Comment:
Add an NPM script to start up the http server so that you can use the app. We now need a HTTP server because we are using the requireJS text plugin to load the html. That plugin will not work on the "file://" protocol (i.e. when you open a html file in chrome) due to security restrictions that browsers have. It requires that the file is hosted on a web server (i.e. 127.0.0.1 or localhost or something).